org.signal.Signal-Beta.json

```diff
@@ -0,0 +1,69 @@
+{
+    "app-id": "org.signal.Signal-Beta",
+    "base": "org.electronjs.Electron2.BaseApp",
+    "base-version": "19.08",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "19.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "signal",
+    "separate-locales": false,
+    "rename-desktop-file": "signal-desktop-beta.desktop",
+    "rename-icon": "signal-desktop-beta",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=pulseaudio",
+        "--share=network",
+        "--device=dri",
+        "--filesystem=home",
+        "--talk-name=org.freedesktop.Notifications",
+        "--talk-name=org.kde.StatusNotifierWatcher",
+        "--own-name=org.kde.StatusNotifierItem-2-1"
+    ],
+    "modules": [
+        {
+            "name": "signal-desktop-beta",
+            "buildsystem": "simple",
+            "build-commands": [
+                "ar x signal-desktop*.deb",
+                "rm -f signal-desktop*.deb",
+                "tar xf data.tar.xz",
+                "rm -f control.tar.gz data.tar.xz debian-binary",
+                "cp -r usr/* opt/* /app",
+                "rm -rf /app/share/icons/hicolor/1024x1024/",
+                "rm '/app/Signal Beta/chrome-sandbox'",
+                "chmod -R a-s,go+rX,go-w '/app/Signal Beta'",
```

Comment:
I think all `build-commands` lines before this one can be simplified with:

```shell
ar -o -x signal-desktop-beta_amd64.deb
tar --extract --file=data.tar.xz
cp --preserve --recursive opt/* usr/* /app"
rm --force --recursive /app/share/icons/hicolor/1024x1024/ '/app/Signal Beta/chrome-sandbox'
```

Also, I would change `/app/Signal Beta` for something like `/app/Signal_Beta`, so there is no need to quote the path and/or escape spaces.